### DISCUSSION TITLE
Is it possible to run 2 directus instances vs the same database?

### DISCUSSION THREAD

**Discussion by faridsaud** [directus/directus#10350] [Archive]
For a high availability setup I was thinking on running 2 directus servers which are connected to the same database. Would this result in any kind of side effects?

**Comment by rijkvanzanten** [comment]
Yes, be aware of anything that is stateful in your current configuration. The main culprits are file-storage and cache/rate-limiting. Once you start scaling up the number of Directus instances, I highly recommend using something like S3 for the file storage, and a Redis instance for cache/rate-limits

**Comment by faridsaud** [comment]
Thank you @rijkvanzanten !
